### ISSUE TITLE
Column aliases not working

### ISSUE THREAD

**Issue by mazedlx** [lorisleiva/laravel-search-string#3]
Hi!

First and foremost: thanks for the great package! I'm having trouble using the column aliases with my setup:

__my model__

```php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;
use Lorisleiva\LaravelSearchString\Concerns\SearchString;

class Contact extends Model
{
    use SearchString;

    protected $searchStringColumns = [
        'zip' => 'plz',
        'city' => 'ort',
    ];
}
```

my __search query__:
```php
<?php
Route::get('/ss', function () {
    return Contact::usingSearchString('plz:1220')->get();
});
```

I get the following __error__:

```
Illuminate \ Database \ QueryException (42S22)
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'plz' in 'where clause' (SQL: select * from `contacts` where `plz` > 1220)
```

If I change the query to `'zip:1220'` everything works as expected. Am I missing something? I'd like to provide the column aliases to my users so they can write the queries in their native language.

Thanks!

**Comment by lorisleiva** [comment]
Hi there 👋 

This is clearly a bug. I'm investigating this.

Thanks for reporting this issue. 👍

**Comment by lorisleiva** [comment]
@mazedlx Okay so I have corrected the bug and added some tests to ensure it's all fixed.

I've published a new release `v0.1.3` containing the fix. Could you please upgrade to the latest version and let me know if everything works for you?

Let me know. 🍀

**Comment by mazedlx** [comment]
Perfect! That fixed it! Thanks again!
